### ISSUE TITLE
services.mirakurun: access to smartcards denied by policykit

### ISSUE THREAD

**Issue by wakira** [NixOS/nixpkgs#122039]
**Describe the bug**
After #97440, pcscd is compiled with policykit support and the default policy allows access only from active session.

This breaks the mirakurun module causing arib-b25 decoding to fail.

systemd journal of pcscd.service
```
pcscd[3910]: 00000000 auth.c:137:IsClientAuthorized() Process 3909 (user: 996) is NOT authorized for action: access_pcsc
pcscd[3910]: 00000070 winscard_svc.c:335:ContextThread() Rejected unauthorized PC/SC client
...
```

I was able to work around this issue by
```
  security.polkit.extraConfig = ''
      polkit.addRule(function(action, subject) {
        if ((action.id == "org.debian.pcsc-lite.access_pcsc" ||
          action.id == "org.debian.pcsc-lite.access_card") &&
          subject.user == "mirakurun") {
          return polkit.Result.YES;
        }
      });
  '';
```

I think we should add that to the mirakurun module.

**Notify maintainers**
@midchildan

**Comment by midchildan** [comment]
@wakira Just in case, could you check if #121246 solves your problem? I unfortunately don't have the means of testing pcscd at the moment.

**Comment by wakira** [comment]
@midchildan I have tested. #121246 was to fix another issue where policykit actions did not get registered at all.

In this case, actions are registered but authorizations are not given to the mirakurun user by default.
